### PR TITLE
flake8 + coverage + black

### PR DESCRIPTION
**Pull request by nvictus**
Applied [black](https://github.com/psf/black) to codebase to make everything pep8-compliant.

Added config files for:
* [flake8](http://flake8.pycqa.org/en/latest/) `.flake8`
* [coverage](https://coverage.readthedocs.io/en/v4.5.x/) `.coveragerc`
* pytest  `pytest.ini` (to run [pytest-cov](https://pytest-cov.readthedocs.io/en/latest/) and [pytest-flake8](https://github.com/tholo/pytest-flake8) plugins)

The pytest plugins will run locally and in CI and will complain when code checked in isn't pep8.

### CODE REVIEW

File: cooltools/cli/call_dots.py [hunk 1]
```diff
@@ -1,172 +1,184 @@
...
     help="Name of value column in EXPECTED_PATH",
     type=str,
-    default='balanced.avg',
+    default="balanced.avg",
     show_default=True,
-    )
+)
 @click.option(
-    '--balancing-weight-name',
+    "--balancing-weight-name",
```
Comment (sergpolly) [inline cooltools/cli/call_dots.py]: somehow this one was reverted - i've made it to be `--weight-name`  just before - now it's back to `--balancing...`
did you start of the latest version with the black-tools ?

File: cooltools/cli/compute_expected.py [hunk 2]
```diff
@@ -11,56 +10,56 @@
...
-    )
+)
 @click.option(
-    '--weight-name',
+    "--weight-name",
     help="Use balancing weight with this name.",
     type=str,
-    default='weight',
-    show_default=True)
+    default="weight",
```
Comment (sergpolly) [inline cooltools/cli/compute_expected.py]: a bunch of stuff dissappeared from `compute_expected` as well ...

File: cooltools/cli/call_dots.py (hunk 1 above)
Comment (nvictus) [reply cooltools/cli/call_dots.py]: yes, i stupidly didn't merge your PR before making this branch!

Comment (sergpolly) [reply cooltools/cli/call_dots.py]: did you do a lot of manual editing after all? i'm wondering what's easier to fix

Comment (sergpolly) [reply cooltools/cli/call_dots.py]: resolve conflicts or start over with black/etc

Comment (nvictus) [reply cooltools/cli/call_dots.py]: i ended up doing a bit of both

**Pull request merged**